src/Pact/Eval.hs

```diff
@@ -65,6 +65,7 @@ import Control.Monad.State.Strict
 import Control.Monad.Reader
 import Unsafe.Coerce
 import Data.Aeson (Value)
+import Text.PrettyPrint.ANSI.Leijen (Pretty(pretty), Doc, dquotes, line, text, parens)
```

Comment:
Might suggest we export these (or some other subset) from Util (which is supposed to function as a Pact prelude (see Pact.Types.Lang which exports `Text,pack,unpack` -- which if we decide this is a good thing should move to Util)